### PR TITLE
Move FlorisCopyToClipboardActivity to `ime.clipboard`

### PR DESCRIPTION
**Pull request by lm41**
This PR moves the FlorisCopyToClipboardActivity from `dev/patrickgold/florisboard/FlorisCopyToClipboardActivity.kt`  to `dev/patrickgold/florisboard/ime/clipboard/FlorisCopyToClipboardActivity.kt`

### CODE REVIEW

**Review by patrickgold — CHANGES_REQUESTED** [review]
thanks for your changes. something isn't right with the Bitmap change:

image

also is it intended that the copied image is left-aligned?

**Review by patrickgold — APPROVED** [review]

**Comment by Klanky42** [comment]
Sorry if this is a silly question, but what does the ime.clipboard do?
I'm concerned about the privacy aspect bc I use the clipboard alot, pasting passwords and so on.
I use FlorisBoard not only for the looks and functions but it's privacy-respecting clipboard is one of the main usage (love the fw/bc buttons btw)
Sorry for being paranoid.
(Wrote this on a Pixel 7 with GrapheneOs and Vanadium browser)

**Comment by lm41** [comment]
`dev.patrickgold.florisboard.ime.clipboard` (`ime.clipboard`)  is the package that contains everything that has to do with the clipboard. The `CopyToClipboardActivity.kt` was mistakenly in the toplevel package `dev.patrickgold.florisboard` and it was moved to the right place with this pr

**Comment by patrickgold** [comment]
As lm41 already said this PR was just a code maintenance update and moved the activity to the correct package. The activity in question allows you to manually import an image to the FlorisBoard-intern clipboard by using the Android share menu and then sharing it to FlorisBoard, so nothing privacy-concerning at all, just a convenience if an app does not support images in clipboards yet

**Comment by Klanky42** [comment]
Thanks @lm41 for the explaination, now I can sleep again 😁
Can you please, in laymanterms, confirm that FlorisBoard is privacy respecting regards clipboard use.
I use the following settings: 
Screenshot_20240823-182428
TIA
And thanks for all the hardwork, as soon as my regular paychecks arrive in my new country, I will start donating to projects I use and enjoy.

**Comment by Klanky42** [comment]
And of course a big thank you to @patrickgold !

**Comment by lm41** [comment]
> Can you please, in laymanterms, confirm that FlorisBoard is privacy respecting regards clipboard use.
